src/scripts/main.js

```diff
@@ -1 +1,14 @@
 'use strict';
+
+const smallImages = document.querySelectorAll('.list-item__link');
+const largeImg = document.querySelector('#largeImg');
+
+smallImages.forEach((link) => {
+  link.addEventListener('click', (e) => {
+    e.preventDefault();
+
+    const srcLink = e.target.src.replace('http://localhost:8080', '');
```

Comment:
The `srcLink` is being set by replacing 'http://localhost:8080' in the `e.target.src`. This assumes that the `src` attribute of the clicked element contains this base URL. If the `src` attribute does not contain 'http://localhost:8080', the replacement will not work as intended. Consider checking if the `src` contains this string before replacing, or ensure that all `src` attributes are formatted consistently.